mautic/mautic
# If a user tries to upload a file to import contacts and something goes wrong, the user gets a message

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [no]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
1. Sometimes CSV file upload fails silently. This PR adds a flash message to inform. 
2. When the user tries to upload a large contact CSV file (file is larger than the `upload_max_filesize`), the upload will fail silently in the default dev env config. The introduction text specifies the max amount to upload. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the import contact page. The max amount to upload is here.
   ![image](https://user-images.githubusercontent.com/96085911/211287559-4c1fd526-2a42-49d3-be72-0c284770a6c7.png)
3. Try uploading a larger file. The flash message is displayed.
  ![image](https://user-images.githubusercontent.com/96085911/211287969-f72311cd-38d2-4900-a39b-55346d00840a.png)
